src/data/crops/grape.js

```diff
@@ -0,0 +1,126 @@
+/** @typedef {import("../../index").farmhand.item} farmhand.item */
+
+import { crop, fromSeed } from '../crop'
+import { cropLifeStage, cropType } from '../../enums'
+
+const { SEED, GROWING } = cropLifeStage
+
+/**
+ * @property farmhand.module:items.grapeSeed
+ * @type {farmhand.item}
+ */
+export const grapeSeed = crop({
+  cropType: cropType.GRAPE,
+  cropTimetable: {
+    [SEED]: 3,
+    [GROWING]: 4,
+  },
+  growsInto: [
+    'grape-chardonnay',
+    'grape-sauvignon-blanc',
+    'grape-pinot-blanc',
+    'grape-muscat',
+    'grape-riesling',
+    'grape-merlot',
+    'grape-cabernet-sauvignon',
+    'grape-syrah',
+    'grape-tempranillo',
+    'grape-nebbiolo',
```

Comment:
this is really neat to think of how many varieties can exist, but i can't help but wonder if this is too many to add. i feel like 3 or 4 would be a pretty good amount, but it's ultimately up to you

Reply:
That's a fair point. 10 varieties is a little overwhelming now that you mention it! I decided to halve the varieties to 5 total in https://github.com/jeremyckahn/farmhand/commit/58a5d9a2fc2c7b366bc2cda7bf726dc94e5c7466. The unused varieties are commented out for now so we can easily add them back in down the line. 🙂